packages/react-router5/types/BaseLink.d.ts

```diff
@@ -54,7 +54,7 @@ declare class BaseLink extends Component<BaseLinkProps, BaseLinkState> {
             style?: React.CSSProperties
             tabIndex?: number
             title?: string
-            inputMode?: string
+            inputMode?: 'none' | 'text' | 'tel' | 'url' | 'email' | 'numeric' | 'decimal' | 'search'
```

Comment:
Is there a way to import this type from React rather than maintaining it manually?

Reply:
could it be ? `inputMode?: React.HTMLAttributes<React.HTMLElement>`

Reply:
Ok find a way: remove `inputMode` declaration and instead pick it from React

```ts
    render(): React.DetailedReactHTMLElement<
        Pick<React.HTMLAttributes<HTMLElement>, 'inputMode'> & {
            href: any
            className: string
     ...
```

Reply:
And I think a lot of attributes could be picked from `React`

Reply:
Another perspective: Do we need to be as much accurate as we are? In which case this is useful for the consumer of this component to have those informations?

I suggest to replace `React.DetailedReactHtmlElement<.......>`  with `React.ReactNode`.

Reply:
Feedback updated. I agreed with @titouancreach suggestion but opted for a smaller change for now.